SWLOR.Game.Server/Feature/UsePerkFeat.cs

```diff
@@ -275,7 +275,12 @@ void CompleteActivation(string id, float abilityRecastDelay)
 
             if (GetIsPC(activator) && !Space.IsPlayerInSpaceMode(activator))
             {
+                ApplyEffectToObject(DurationType.Temporary, EffectSlow(), activator, 1.5f);
                 ApplyEffectToObject(DurationType.Temporary, EffectCutsceneImmobilize(), activator, 1.5f);
+                DelayCommand(1.5f, () =>
```

Comment:
Let's make this 1.6f to ensure it happens after the other effects are intended to expire.